packages/mdc-top-app-bar/mdc-top-app-bar.scss

```diff
@@ -36,13 +37,12 @@
     position: relative;
     box-sizing: border-box;
     width: 100%;
-    height: auto;
-    min-height: $mdc-top-app-bar-row-height;
+    height: $mdc-top-app-bar-row-height;
```

Comment:
Curious why we had height + min-height earlier but now don't need it?

Reply:
It was brought over from `toolbar`. I *think* it was used for flexible toolbars. I'll leave it in so we don't have to change it all when we get the flexible toolbar moved.